src/components/InputCode/__tests__/inputCode.spec.tsx

```diff
@@ -0,0 +1,36 @@
+import 'react-native';
+import React from 'react';
+import TestRenderer from 'react-test-renderer';
+import InputCode from '../index';
+import handlePasteCode from '../helpers/getCodeFromClipboard';
+
+describe('<InputCode/>', () => {
+    const numberOfDigits = 6;
+    it('should create an input Code by the number of digits provided', () => {
+        const testRenderer = TestRenderer.create(
+            <InputCode digits={numberOfDigits} />,
+        );
+        const testInstance = testRenderer.root;
+        expect(testInstance.findAllByType('TextInput').length).toEqual(
+            numberOfDigits,
+        );
+    });
+
+    it('should recognize code in a string', () => {
```

Comment:
since what you are testing is the handlePasteCode function we should put its tests in another file:
src/components/InputCode/helpers/__tests__/getCodeFromClipboard.ts